src/components/Wrapper/style.scss

```diff
@@ -1,26 +1,30 @@
 .wrapper {
   position: relative;
+  display: flex;
+  height: 100vh;
```

Comment:
100vh and overflow hidden might caused some problems if there will be more text/ components in the view

Reply:
This is also fixed on another branch